DQMOffline/Trigger/python/METMonitor_cfi.py

```diff
@@ -0,0 +1,32 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.metMonitoring_cfi import metMonitoring
```

Comment:
It is auto-generated via fillDescriptions:
https://cmssdt.cern.ch/lxr/source/DQMOffline/Trigger/plugins/METMonitor.cc#0273